model_compression_toolkit/exporter/model_wrapper/fw_agnostic/get_inferable_quantizers.py

```diff
@@ -32,18 +32,25 @@ def get_inferable_quantizers(node: BaseNode,
     Returns:
         weight_quantizers: A dictionary between a weight's name to its quantizer.
         activation_quantizers: A list of activations quantization, one for each layer output.
+        weight_values: A dictionary between a weight's name to its value. Relevant for positional weights only.
```

Comment:
Fix comment. Also, why not set the default value of attributes_name to None instead of a mutable list?